### ISSUE TITLE
RuntimeWarning: DateTimeField Event.timestamp received a naive datetime 

### ISSUE THREAD

**Issue by andreybldup** [PostHog/posthog#709]
**Describe the bug**
After updating to the latest docker container, timestamps are arriving from the future

**To Reproduce**
1. Run Posthog 1.2.0 (I believe) via docker-compose
2. Upgrade to 1.3.0

**Expected behavior**
Expecting no changes in terms of timestamp handling from previous version.

**Screenshots**
![posthogtimestamps](https://user-images.githubusercontent.com/57500678/80916583-974f0280-8d27-11ea-832e-d4fcfd57b188.png)

**Desktop (optional: please complete the following information):**
 - OS:
 - Browser: 
 - Version: 

**Smartphone (optional: please complete the following information):**
 - Device: 
 - OS: 
 - Browser 
 - Version 

**Additional context**
```
web_1    | /usr/local/lib/python3.8/site-packages/django/db/models/fields/__init__.py:1307: RuntimeWarning: DateTimeField Event.timestamp received a naive datetime (2020-04-03 00:00:00) while time zone support is active.
web_1    |   warnings.warn("DateTimeField %s.%s received a naive datetime "

```
I also get some ``` duplicate key value violates unique constraint "unique distinct_id for team"``` errors, but I'm pretty sure those are unrelated

**Comment by timgl** [comment]
Hey @andreybldup, we merged in some fixes for this after we updated to 1.3.0. Could you try pulling the :latest image?

**Comment by andreybldup** [comment]
I updated, looks like it's good now. Thanks!
